src/documents/ChangeApplierAPI.md

```diff
@@ -248,7 +367,7 @@ at the time the component is constructed. Note that programmatically attached li
 The listener is notified after the change (or set of coordinated changes) has already been applied to the model - it is too late to affect this process and so this event is not _preventable_. The signature for these listeners is
```

Comment:
After a bit more thinking, doe "it is too late to affect this process" mean `excludeSource` and `includeSource` won't take effect for these listeners?